### PR TITLE
Overlay should correctly restore affecting size css attributes after fullscreen option change (T937118)

### PR DESCRIPTION
**Pull request by ksercs**
…

### CODE REVIEW

**Review by DokaRus — DISMISSED** [review]

File: testing/tests/DevExpress.ui.widgets/overlay.tests.js [hunk 1]
```diff
@@ -2161,6 +2161,29 @@ testModule('close on target scroll', moduleConfig, () => {
...
+
+        const overlay = $('#overlay').dxOverlay({
+            container: $container,
+            visible: true
+        }).dxOverlay('instance');
+
+        let wrapperElement = overlay.$content().parent().get(0);
+
+        assert.strictEqual(wrapperElement.style.width, '100px', 'width is correct');
+        assert.strictEqual(wrapperElement.style.height, '100px', 'height is correct');
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets/overlay.tests.js]: Do we have a test that already tests this scenario?

```suggestion

```

File: testing/tests/DevExpress.ui.widgets/overlay.tests.js [hunk 2]
```diff
@@ -2161,6 +2161,29 @@ testModule('close on target scroll', moduleConfig, () => {
...
+
+        overlay.option('container', null);
+        wrapperElement = overlay.$content().parent().get(0);
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets/overlay.tests.js]: Why do we need to acquire a wrapper element once again?

File: testing/tests/DevExpress.ui.widgets/popup.tests.js [hunk 3]
```diff
@@ -503,6 +503,28 @@ QUnit.module('dimensions', {
...
 
+    QUnit.test('affecting size css atrrtibutes should be restored after fullScreen set to true', function(assert) {
+        const instance = $('#popup').dxPopup({
+            maxHeight: 100,
+            minHeight: 100,
+            minWidth: 100,
+            maxWidth: 100
+        }).dxPopup('instance');
+
+        instance.show();
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets/popup.tests.js]: Can we use `visible: true` instead?

File: testing/tests/DevExpress.ui.widgets/popup.tests.js [hunk 4]
```diff
@@ -503,6 +503,28 @@ QUnit.module('dimensions', {
...
+        const overlayContentElement = instance.$content().parent().get(0);
+        const affectingSizeOptions = ['minWidth', 'maxWidth', 'minHeight', 'maxHeight'];
+
+        affectingSizeOptions.forEach(attr => {
+            assert.strictEqual(overlayContentElement.style[attr], '100px', 'css attr value is correct');
+        });
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets/popup.tests.js]: It looks like another test case to me.
```suggestion
```

File: testing/tests/DevExpress.ui.widgets/overlay.tests.js (hunk 1 above)
Comment (ksercs) [reply testing/tests/DevExpress.ui.widgets/overlay.tests.js]: no, but I separated this test to 2 different

File: testing/tests/DevExpress.ui.widgets/overlay.tests.js (hunk 2 above)
Comment (ksercs) [reply testing/tests/DevExpress.ui.widgets/overlay.tests.js]: I separated this test to 2 different, so there is no necessity more

File: testing/tests/DevExpress.ui.widgets/popup.tests.js (hunk 3 above)
Comment (ksercs) [reply testing/tests/DevExpress.ui.widgets/popup.tests.js]: yeap

File: testing/tests/DevExpress.ui.widgets/popup.tests.js (hunk 4 above)
Comment (ksercs) [reply testing/tests/DevExpress.ui.widgets/popup.tests.js]: fixed

**Review by DokaRus — APPROVED** [review]

**Review by San4es — APPROVED** [review]

**Pull request merged**